test/files/mock-insights

```diff
@@ -9,51 +9,95 @@
 #
 # You need these in your insights-client.conf:
 #
-# gpg=False
 # auto_config=False
 # base_url=127.0.0.1:8888/r/insights
 # insecure_connection=True
+# username=admin
+# password=foobar
 
 from http.server import *
 import json
+import re
 
 systems = { }
 
 class handler(BaseHTTPRequestHandler):
+    def match(self, p):
+        return re.fullmatch(p, self.path)
+
     def do_GET(self):
-        if self.path == "/r/insights/":
+
+        m = self.match("/r/insights")
+        if m:
             self.send_response(200)
             self.end_headers()
             self.wfile.write(b"lub-dup")
-        elif self.path.endswith("/static/core/insights-core.egg"):
+            return
+
+        m = self.match("/r/insights/v1/static/core/insights-core.egg")
+        if m:
             self.send_response(304)
             self.end_headers()
-        elif self.path.endswith("/static/uploader.v2.json"):
+            return
+
+        m = self.match("/r/insights/v1/static/uploader.v2.json")
+        if m:
             # This is not a valid response and will cause the client
             # to fall back to the builtin rules.
             self.send_response(200)
             self.end_headers()
             self.wfile.write(b"{ }\n")
-        elif "/systems/" in self.path:
-            machine_id = self.path.split("/")[-1]
+            return
```

Comment:
> This mock insights server does not work as expected, because it doesn't return anything for request `uploader.v2.json.asc`

That is probably because `gpg=False` is missing in the config.

Reply:
I have updated mock-insights to the latest version, which had some improved comments.